plenum/common/transactions.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum, unique
+
+
+@unique
+class PlenumTransactions(Enum):
+    #  These numeric constants CANNOT be changed once they have been used,
+    #  because that would break backwards compatibility with the ledger
+    NODE = "0"
```

Comment:
I wonder if we also need a comment saying that these numbers also CANNOT collide with transaction constants in depending components (e.g., sovrin-*). It would be bad if a transaction in sovrin-* had a numeric value of 0 or 1. It would also be bad if it had a value of 2, and then we edited plenum to add a value to the enum that created a collision...